### PR TITLE
Migrate from ebean 8.1.1 to ebean 10.1.3

### PR DESCRIPTION
**Pull request by PierreAdam**
- Variabilize the version of Ebean in the SBT file.
- Change the dependencies to the new io.ebean groupId.
- Change artifactId avaje-ebeanorm-mavenenhancer to the new ebean-maven-plugin
- Change the imports

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <ebean.agent.version>10.1.2</ebean.agent.version>
```
Comment (thibaultmeyer) [inline pom.xml]: Ebean version is not necessary the same as the Ebean agent version. Could you create `ebean.version` variable set to _10.1.3_ please

File: src/main/java/com/zero_x_baadf00d/play/module/aws/s3/ebean/BaseS3FileModel.java [hunk 2]
```diff
@@ -41,6 +41,7 @@
  * the implementation of a model using the Amazon S3 plugin.
  *
  * @author Thibault Meyer
+ * @author Pierre Adam
  * @version 16.06.28
```
Comment (thibaultmeyer) [inline src/main/java/com/zero_x_baadf00d/play/module/aws/s3/ebean/BaseS3FileModel.java]: Could you update `@version` tag to _17.01.03_

File: src/main/java/com/zero_x_baadf00d/play/module/aws/s3/ebean/S3FileModel.java [hunk 3]
```diff
@@ -33,6 +33,7 @@
  * This Ebean model represent a file stored on Amazon S3.
  *
  * @author Thibault Meyer
+ * @author Pierre Adam
  * @version 16.03.19
```
Comment (thibaultmeyer) [inline src/main/java/com/zero_x_baadf00d/play/module/aws/s3/ebean/S3FileModel.java]: Could you update `@version` tag to _17.01.03_

**Review by thibaultmeyer — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (thibaultmeyer) [inline pom.xml]: ebean.agent.version -> ebean.enhancer.version

**Review by thibaultmeyer — CHANGES_REQUESTED** [review]

**Comment by PierreAdam** [comment]
Done

**Pull request merged**